package.json

```diff
@@ -50,6 +51,8 @@
     "eslint-plugin-import": "2.9.0",
     "eslint-plugin-jsx-a11y": "6.0.3",
     "eslint-plugin-react": "7.7.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.1",
```

Comment:
In TDB we use:
"react": "16.3.2",
"react-dom": "16.3.2"

Let's use the same here too

Reply:
it doesn't matter. npm will automatically force this repo to use `v16.3.2` because i set it to `^16.2.0`. that caret symbol tells npm to use any version at 16.2.0 or above but below version 17